src/cloudimagedirectory/filter/filter.py

```diff
@@ -23,6 +36,13 @@ def FilterImageByLatestUpdate(latestDate: pd.Timestamp) -> Callable:  # type: ig
     print(f"filter images by latest date: {latestDate}")
     latestDate = latestDate.replace(tzinfo=pytz.UTC)
 
+    filtered_image_by_latest_update_counter = meter.create_counter(
+        name="filtered_image_by_latest_update_counter",
+        description="Counts the number of filtered images by latest update",
+        unit="1",
+    )
+    # TODO: Call method every time the lambda skips an element.
+    # filtered_image_by_latest_update_counter.add(1, {"latest_date", latestDate})
```

Comment:
Whats the best way to call the counter add method every time an item is filtered?
```python
return lambda data: [d for d in data if d.content is not None and get_utc_datetime(d.content["date"]) > latestDate]
```